alacritty/src/config/window.rs

```diff
@@ -65,6 +68,7 @@ pub struct WindowConfig {
 impl Default for WindowConfig {
     fn default() -> Self {
         Self {
+            blur: false,
```

Comment:
```suggestion
            blur: Default::default(),
```

Can move that to the bottom too.